src/io/local_filesys.cc

```diff
@@ -69,13 +69,22 @@ class FileStream : public SeekStream {
 
 FileInfo LocalFileSystem::GetPathInfo(const URI &path) {
   struct stat sb;
+  FileInfo ret;
+  ret.path = path;
   if (stat(path.name.c_str(), &sb) == -1) {
     int errsv = errno;
-    LOG(FATAL) << "LocalFileSystem.GetPathInfo " << path.name
-               << " Error:" << strerror(errsv);
+    // If lstat succeeds where stat failed, assume a problematic
+    // symlink and treat this as if it were a 0-length file.
+    if (lstat(path.name.c_str(), &sb) == 0) {
```

Comment:
@DickJC123 This breaks MXNet CI tests on Windows. Please see: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-9869/6/pipeline